px-helm-mongo/3.create-volume.md

```diff
@@ -0,0 +1,25 @@
+In this step, we will create a Portworx volume (PVC) for Mongo.
+
+### Step: Create StorageClass
+
+Take a look at the StorageClass definition for Portworx: ```cat px-ha-sc.yaml```{{execute T1}}
+
+The parameters are declarative policies for your storage volume. See [here](https://docs.portworx.com/manage/volumes.html) for a full list of supported parameters.
```

Comment:
The URL is incorrect. It should be https://docs.portworx.com/scheduler/kubernetes/dynamic-provisioning.html

Reply:
I fixed it